core/src/main/scala/org/apache/spark/sql/SnappyStrategies.scala

```diff
@@ -124,7 +122,8 @@ private[sql] trait SnappyStrategies {
           s"Expected one of ${Constant.ALLOWED_JOIN_TYPE_HINTS}")
     }
 
-    def apply(plan: LogicalPlan): Seq[SparkPlan] = if (isDisabled || session.disableHashJoin) {
+    def apply(plan: LogicalPlan): Seq[SparkPlan] = if (isDisabled ||
+        snappySession.disableHashJoin) {
```

Comment:
unnecessary line break or was it exceeding 100 chars?

Reply:
it was exceeding 100 chars.